README.md

```diff
@@ -1,67 +1,69 @@
+If you wish to run a company that employs non-FAC staff, then you will need to get agreement from the FAC directors, who must be satisfied that the arrangement is of benefit to FAC. 
 
-**I am now employed elsewhere. Do I have to give up my membership?**    
-Not at all. As long as you are prepared to abide by our membership rules, you will always continue to be welcome as a member. Wherever you are, we will expect you to help with FAC programmes or to set up events, meetup groups and study programmes under the FAC banner (and jointly with your own company or employer, where appropriate).
+**If I am employed elsewhere, does that mean I'm no longer a part of FAC?**    
+Not at all. As long as you are prepared to help out, you will always continue to be welcome. Wherever you are, we always want you to help with FAC programmes or to set up events, meetup groups and study programmes under the FAC banner (and jointly with your own company or employer, if you like).
 
-**I live and work abroad. Do I still need to refer prospective employers to FAC?**    
+**I am moving abroad. Do I still need to refer prospective employers to FAC?**    
 Not unless we have a presence in the city where you work, but do please set up events, meetup groups and study programmes under the FAC banner and keep us posted.
 
-**Will FAC help me find work?**    
-FAC will support you in your job hunt and mentor you through interviews, but we do not currently have the resources to hire a dedicated recruiter, so in the short-term members will need to act as their own recruiters, but when resources allow, then we may well employ somebody. Also, as we build up a network of partner companies and agency relationships, active recruitment for recent graduates may become less of an issue.
+**Will FAC help me find work quickly?**    
+FAC will try to support you in your job hunt and mentor you through interviews, but we do not currently have the resources to hire a dedicated recruiter. As we build up a network of partner companies and agency relationships, active recruitment for recent graduates may become less of an issue, but in the short term graduates may have less support than we would like.
+
+**If I tell FAC about a job I want to apply for, will FAC tell other graduates about the job and increase competition for it?**  
+We think the fairest thing would be to only advertise this job after a suitable period of time has passed. 
 
-**This makes me uneasy. I don't want to involve FAC in my conversations with prospective employers.**    
-We understand this. FAC membership may not be the right choice for you.
+**This rule makes me uneasy. I don't want to involve FAC in my conversations with prospective employers.**    
+We understand this, but as our primary source of income, this is a condition of joining the programme.
```

Comment:
Good point. I think that if we view this document as a condition of acceptance rather than a charter, that issue becomes simpler - we can just change the terms of acceptance as circumstances change.

It would be more of an issue if we wanted this document to be a full and unwavering exposition of our principles